build-system/tasks/storybook/index.js

```diff
@@ -58,12 +66,13 @@ function launchEnv(env) {
       isCiBuild() ? '--ci' : '',
     ].join(' '),
     {cwd: __dirname, stdio: 'inherit'}
-  );
+  ).on('error', () => {
+    throwError('Launch failed');
```

Comment:
Can we include the error message?

Reply:
Error message and call stack are already in the logs. This merely sends a failure reason to the `gulp` process. See screenshot in the PR description.